docs/docs/snippets/supported-types.mdx

```diff
@@ -13,6 +13,25 @@ Here's a list of all the types that can be represented in BAML:
 * `string`
 * `null`
 
+## Literal Types
```

Comment:
can you mark what version of BAML supports this?

Reply:
Done @hellovai